react/components/EXPERIMENTAL_Table/hooks/useTableState.ts

```diff
@@ -0,0 +1,26 @@
+import { useMemo } from 'react'
+
+import { calculateTableHeight } from '../util'
+import constants from '../constants'
+
+interface Input {
+  columns: ColumnObject
+  items: Array<any>
+}
+
+const useTableState = ({ columns, items }: Input): TableState => {
+  const isEmpty = useMemo(() => Object.keys(columns).length === 0, [columns])
```

Comment:
I don't get this. Shouldn't you be checking for items? Is empty is for when the schema defines 0 columns? 🤔

Reply:
Oops! Nice catch, I will add the checking for items. In the future, the checking for columns will also be necessary (hidden fields feature).